SPECS-EXTENDED/osinfo-db/osinfo-db.spec

```
@@ -0,0 +1,277 @@
+Summary:        osinfo database files
+Name:           osinfo-db
+Version:        20221130
+Release:        2%{?dist}
+License:        LGPLv2+
```

Same comment as `libosinfo`.

```suggestion
License:        LGPL-2.1-or-later
```